test/suites/drift/suite_test.go

```diff
@@ -588,6 +588,10 @@ var _ = Describe("Drift", func() {
 			// Remove the startup taints from the new nodes to initialize them
 			Eventually(func(g Gomega) {
 				g.Expect(env.Client.Get(env.Context, client.ObjectKeyFromObject(nodeTwo), nodeTwo)).To(Succeed())
+				_, found := lo.Find(nodeTwo.Spec.Taints, func(t corev1.Taint) bool {
+					return t.Key == "example.com/another-taint-2" && t.Effect == corev1.TaintEffectPreferNoSchedule
```

Comment:
Can you use `.MatchTaint(corev1.Taint{Key: "example.com/another-taint-2", Effect: corev1.TaintEffectPreferNoSchedule})` here?

Reply:
Same thought